harsohailB/Jambo
# add delete route and bind to item remove button

**Goal of this branch:** Create a delete route and bind it to the "remove item" button accessible to website management

**How to test:**

* I have added a sample test item to test deletion on as the add item feature is not finished yet. Here is the sample item:

```
{
      "id": 16,
      "name": "Test Item",
      "price": 27.99,
      "folderName": "test",
      "thumbnailImage": {
        "imageName": "adult-fox-on-grass-2786709.jpg",
        "color": "None"
      },
      "colors": "",
      "sizes": "",
      "description": "",
      "images": []
    }
```

1. Start the front-end and back-end of the application
2. Navigate to the catalog page and then the test item
3. Click the remove button
4. Check the ```src\assets\catalog\inventoryData.json``` file to see the changes
5. Visit the catalog page again to see the changes
